### DISCUSSION TITLE
Displaying content of a .txt or .doc file

### DISCUSSION THREAD

**Discussion by olazzzzz** [freeplane/freeplane#1522] [Ideas]
In my personal use of Freeplane, there are moments in which I would like to take note of something to later include it in a mindmap but I happen to be outside without my laptop. 
So I end up taking notes in some other software using my Android and then transposing the content to Freeplane.
Question: would it not be possible to have a .txt or .doc file being displayed in the mindmap? In this way, all I would have to do would be to edit the content with the smartphone and it would automatically show on the mindmap.

Thank you, greetings

**Comment by euu2021** [comment]
Related discussions: 
[#478 ](https://github.com/freeplane/freeplane/discussions/478#discussioncomment-3093739)
#1219 

I'm still doing what I said in the topic above: create the node in Todoist; and then copy and paste into Freeplane. I do that everyday, and it works well.

I know that this add-on does some sort of sync with a txt, but I never tried it: [GTD Syn](https://sourceforge.net/projects/gtdsync/)c by  haaihenkie

**Comment by olazzzzz** [comment]
Thanks @euu2021 for your reply.

In order not to lose myself for weeks in a rabbit whole, I created a .txt file on Dropbox on my smartphone and made a shortcut to it on my Tasks.mm. Of course I still have to open the file and copy/paste it, but in this way I don't need extra apps installed. It is maybe not perfect but this will work for the moment.
